packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -74,6 +75,11 @@ type ChoicePropsWithForwardRef = {|
     ...WithForwardRef,
 |};
 
+let id = 0;
```

Comment:
A comment here would be nice linking to the fix ticket kevin is working on

Reply:
Added! :)